composer.json

```diff
@@ -26,7 +26,7 @@
         "symfony/form": "~2.1,!=2.5.0",
         "symfony/validator": "~2.1,!=2.5.0",
         "symfony/translation": "~2.1,!=2.5.0",
-        "doctrine/orm": "~2.2,>=2.2.3"
+        "doctrine/orm": "~2.2,>=2.2.3,<2.5"
```

Comment:
Hm, I did like that because this is how Symfony does it. https://github.com/symfony/symfony-standard/blob/2.7/composer.json#L12

I improved the .travis.yml instead.. lets see if it works.